server/build.gradle

```diff
@@ -34,39 +34,39 @@ publishing {
 
 archivesBaseName = 'elasticsearch'
 
-// we want to keep the JDKs in our IDEs set to JDK 8 until minimum JDK is bumped to 9 so we do not include this source set in our IDEs
+// we want to keep the JDKs in our IDEs set to JDK 11 until minimum JDK is bumped to 17 so we do not include this source set in our IDEs
```

Comment:
Why aren't we removing the additional source set just like we did for libs ? 
Are we anticipating a need to to MR jars for java 12 ?

Reply:
Yeah, I'm anticipating that indeed.